libraries/node-core-library/src/IPackageJson.ts

```diff
@@ -157,6 +157,12 @@ export interface IPackageJsonTsdocConfiguration {
    * A table of script hooks that a package manager or build tool may invoke.
    */
   scripts?: IPackageJsonScriptTable;
+
+  /**
+   * A table of package version resolutions. This feature is only available in
+   * yarn.
+   */
+  resolutions?: Object;
```

Comment:
Can you give this a more specific type?

```suggestion
  resolutions?: { [name: string]: string };
```